app/src/main/res/values/strings.xml

```diff
@@ -531,5 +531,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="images_featured_explanation">Featured pictures are images from highly skilled photographers and illustrators that the Wikimedia Commons community has chosen as some of the highest quality on the site.</string>
   <string name="images_via_nearby_explanation">Images Uploaded via Nearby places are the images which are uploaded by discovering places on the map.</string>
   <string name="thanks_received_explanation" >This feature allows editors to send a Thank you notification to users who make useful edits – by using a small thank link on the history page or diff page.</string>
+    <string name="welcome_do_upload_contentDescription">Examples of good images to upload to Commons</string>
+    <string name="welcome_dont_upload_contentDescription">Examples of images not to upload</string>
```

Comment:
Congratz to myself I could came so far of code reading:) Is this intentionally welcome_dont_upload_contentDescription instead of welcome_dont_upload_content_description

Reply:
I think you're right, it'd be better as content_description. Will fix later.

Reply:
Once it's fixed, would you be okay with me re-requesting your review?

Reply:
Sure, actually my tests were successful, after this easy fix we can go for merge @domdomegg